src/geo/lng_lat.js

```diff
@@ -42,7 +42,11 @@ class LngLat {
      * wrapped.lng; // = -73.9749
      */
     wrap() {
-        return new LngLat(wrap(this.lng, -180, 180), this.lat);
+        if (this.lng <= -180 || this.lng > 180) {
```

Comment:
Since my changes meant that LngLat.wrap is called a lot more (every camera change due to an interaction), I wanted to ensure that creating the new LngLat object wouldn't impact performance.

Although on further testing, map interaction seems to have no degradation without this check. So I'm not sure whether to leave it in and update the API docs or just remove this change, and leave it as is.

Thoughts?